Postgresql/legos/postgresql_read_query/postgresql_read_query.py

```diff
@@ -15,13 +15,13 @@ class InputSchema(BaseModel):
         title='Read Query',
         description='''
             Read query in Postgresql PREPARE statement format. For eg.
-            SELECT foo FROM table WHERE bar=$1 AND customer=$2.
-            The values for $1 and $2 should be passed in the params field as a tuple.
+            SELECT foo FROM table WHERE bar=%s AND customer=%s.
+            The values for %s and %s should be passed in the params field as a tuple.
         ''')
     params: tuple = Field(
         None,
         title='Parameters',
-        description='Parameters to the query in list format. For eg: [1, 2, "abc"]')
+        description='Parameters to the query in tuple format. For eg: ("abc")')
```

Comment:
@sagar-salvi-unskript PREPARE statements are supposed to be used like this `https://www.postgresql.org/docs/current/sql-prepare.html`. We cant change the behaviour for the user just bcos of the library.
Can we see if this `https://stackoverflow.com/questions/1616106/python-to-postgres-interface-with-real-prepared-statements` works and implement this?